do_cmake.sh

```diff
@@ -27,6 +27,15 @@ if [ "$PYBUILD" = "3" ] ; then
     ARGS="$ARGS -DWITH_PYTHON2=OFF -DWITH_PYTHON3=ON -DMGR_PYTHON_VERSION=3"
 fi
 
+#ARGS="${ARGS} -DCMAKE_BUILD_TYPE=Release"
+args_build_type=$(echo $ARGS | sed 's/^.*\?-DCMAKE_BUILD_TYPE=\(\w\+\).*\?$/\1/')
```

Comment:
@the78mole i'd suggest drop this change. `do_cmake.sh` is designed to be a very thin wrapper around cmake. and it is supposed to work for most developers. if some of us need to fine tune the options. i think we should encourage them to use `cmake` directly for the maximum flexibility.